src/main/webapp/app/list-of-more-feedback-requests/list-of-more-feedback-requests.component.html

```diff
@@ -0,0 +1,121 @@
+<div class="course-info-bar">
```

Comment:
This component looks like an exact copy of the list-of-complaints component.
It also has the same selector as the more-feedback component :thinking: 
I think we don't need this file, but you should double check.

Reply:
You are right. I understand that it was a bad idea to split them into two components